ci/tests/small_fixed_alloc.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include "gc.h"
+#include <cheriintrin.h>
+
+int main(void) 
+{
+    int i;
+    int iter_total = 16*1024; 
```

Comment:
I think this might better be a `#define` constant rather than looking like a variable?

Reply:
Done in [2cc9b31d](https://github.com/capablevms/bdwgc/pull/11/commits/2cc9b31d999e78ceced82eadae95bcd57306d72a)